libs/libs-pomfix/lib/fix.js

```diff
@@ -118,7 +118,8 @@ module.exports = function (load, options) {
   var findDeps = function () {
     var findPom = function (path) {
       if (path.match('features.xml')) {
-        return path.replace(/src\/.+/, 'pom.xml')
+        // NOTE: assumes that no module path contains src except for the src directory
+        return path.replace(/src.+/, 'pom.xml')
```

Comment:
Shouldn't we be more strict here? Otherwise, if the `path` is `some/path/isrcall/more/src/features.xml`, this would produce `some/path/ipom.xml`, which is certainly not something we want.